voc/metric.go

```diff
@@ -25,9 +27,8 @@
 
 package voc
 
-type LogOutput struct {
-	*Functionality
-	Logging []ResourceID `json:"logging"`
-	Call    string       `json:"call"`
-	Value   string       `json:"value"`
+type Metric struct {
```

Comment:
I think this was actually a left over from a MEDINA experiment and can be deleted.

Reply:
I deleted the Metric in the Ontology.